config/management/operational/src/keys.rs

```diff
@@ -20,6 +164,8 @@ struct ExtractKey {
     /// Location to store the key
     #[structopt(long)]
     key_file: PathBuf,
+    #[structopt(long)]
+    encoding: EncodingType,
```

Comment:
hmm... this is a breaking change, can you do a check on the partners repo to see if this function is used anywhere?

Reply:
Eh, I just am making it default to BCS.  It'll be it's main use case anyways